src/ngraph/runtime/gpu/gpu_external_function.cpp

```diff
@@ -458,17 +458,18 @@ void runtime::gpu::GPU_ExternalFunction::emit_temp_mem_pool_allocation(
     {
         m_writer << "// Allocate the memory pool\n";
         // TODO memory pool malloc.
-        m_writer << "void* pool_base_ptr = ngraph::runtime::gpu::invoke_memory_primitive(ctx, "
-                 << m_tensor_memory_buffers->at(current_function->get_name()) << ");\n";
+        m_writer
+            << "char* pool_base_ptr = (char*)ngraph::runtime::gpu::invoke_memory_primitive(ctx, "
```

Comment:
Because we cast it to char* everywhere because you can't do pointer math on a void*. If it is defined as char* then we can remove the cast from everywhere else.